dashboard/src/components/createStyledComponent/createStyledComponent.js

```diff
@@ -0,0 +1,51 @@
+import classnames from "classnames";
+import { withStyles } from "@material-ui/core/styles";
+import {
+  compose,
+  setDisplayName,
+  mapProps,
+  defaultProps,
+  componentFromProp,
+} from "recompose";
+
+let id = 0;
+const getNextId = () => {
+  id += 1;
+  return id;
+};
```

Comment:
This little thing should have it's own happy place in a util dir at this point.

Reply:
for sure 🤔